### PR TITLE
Note about workaround for kubernetes#34566

### PR DESCRIPTION
**Pull request by errordeveloper**
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/1431)

### CODE REVIEW

**Comment by errordeveloper** [comment]
cc @luxas @lukemarsden

**Comment by luxas** [comment]
Happy to merge but:
a) Seems like the "> " indentation doesn't work: http://deploy-preview-1431.kubernetes-io-vnext-staging.netlify.com/docs/getting-started-guides/kubeadm/
b) This could be in a Virtualbox section (we will soon end up with notes for different envs anyway)

**Comment by errordeveloper** [comment]
I should have Linux Foundation CLA now also.

**Comment by luxas** [comment]
LGTM

File: docs/getting-started-guides/kubeadm.md [hunk 1]
```diff
@@ -254,3 +254,9 @@ Please note: `kubeadm` is a work in progress and these limitations will be addre
 1. There is not yet an easy way to generate a `kubeconfig` file which can be used to authenticate to the cluster remotely with `kubectl` on, for example, your workstation.
 
    Workaround: copy the kubelet's `kubeconfig` from the master: use `scp root@<master>:/etc/kubernetes/admin.conf .` and then e.g. `kubectl --kubeconfig ./admin.conf get nodes` from your workstation.
+
+1. If you are using VirtualBox (directly or via Vagrant), you will need to ensure that `hostname -i` returns a routable IP address (i.e. one on the second network interface, not the first one).
+   By default, it doesn't do this and kubelet ends-up using first non-loopback network interface, which is usually NATed.
+   Workaround: Modify `/etc/hosts`, take a look at this [`Vagrantfile`][ubuntu-vagrantfile] for how you this can be achieved.
```
Comment (caseydavenport) [inline docs/getting-started-guides/kubeadm.md]: s/for how you this can be/for how this can be

**Comment by errordeveloper** [comment]
We will have to move this into a special docs for VirtualBox users, there
are too many things to talk about...

**Pull request merged**